pkg/printers/internalversion/printers_test.go

```diff
@@ -828,12 +828,30 @@ func TestPrintNodeStatus(t *testing.T) {
 		{
 			node: api.Node{
 				ObjectMeta: metav1.ObjectMeta{
+					Name:   "foo10",
+					Labels: map[string]string{"node-role.kubernetes.io/master": "", "kubernetes.io/role": "node", "kubeadm.alpha.kubernetes.io/role": "node"},
```

Comment:
Does specifying `["node-role.kubernetes.io/master"] = ""` mean the Node is a master? If so, why is the string left blank? I would expect to see "true" or similar.